Sources/Foundation/CMakeLists.txt

```diff
@@ -178,3 +178,7 @@ endif()
 
 set_property(GLOBAL APPEND PROPERTY Foundation_EXPORTS Foundation)
 _foundation_install_target(Foundation)
+install(FILES
```

Comment:
Should we move this to the swift-corelibs-foundation repo where these modules are defined instead? It seems a bit odd to be installing pieces of swift-foundation from another repository. Also, is this safe to apply to all platforms or should this be conditionalized to Windows/other applicable platforms?

Reply:
The generator expression should evaluate to the `.lib` on Windows, `.tbd` on macOS (if present), and empty string on Linux. Installing it in swift-foundation makes sense, but we will need to copy it here as well I think.

Reply:
Yeah it seems like this repo might need to do this for the swift-corelibs-foundation libraries like `Foundation`/`FoundationXML`/`FoundationNetworking` but it'd be great if we could move the treatment of the swift-foundation libraries to that repo. Perhaps we could just add an extra `install` call to the `_foundation_install_target` function so we can make sure this is appropriately handled for all of our targets?